addons/website/static/src/snippets/s_timeline/000.scss

```diff
@@ -1,18 +1,18 @@
 .s_timeline {
-    .s_timeline_line {
+    .s_timeline_container {
         position: relative;
-        &:before {
-            content: '';
-            display: block !important; // override portal '#wrap .container' value
+        .s_timeline_line {
+            border: 0px solid gray('800');
+            border-right-width: 1px;
             position: absolute;
-            width: 1px;
             top: 0px;
             bottom: 0px;
             left: 50%;
-            background-color: gray('800');
+            z-index: -1;
         }
     }
     .s_timeline_row {
+        position: relative;
```

Comment:
The whole point of 000.scss, 001.scss, etc is to be able to keep old version of snippets in user database working.  If you change the structure of the DOM / the CSS, you have to create a migration script to migrate old snippets of users. And in the case that migration script is not possible, you have to keep the 000.scss file as it is, create a new 001.scss file and increase the version counter on the snippet. In this case, I think you could make a migration script.

Is it required though? To avoid the need of that migration script, maybe you could keep the old snippet structure and CSS and implement the feature with a small trick: instead of background-color of the ::before element here, use `border-color: inherit` and make your editor option control the border-color of current "s_timeline_line" (which is the snippet "container") -> that won't ever affect the "container" element since it has no border (and should never have one) and that will allow to control the ::before line's color without breaking compatibility with snippets already used by customers.

Reply:
That's a nice trick :+1: , I tried this the first time with `snippet_options_border_line_widgets` options, but it seems that both "border-style" & "border-color" options depend on "border-with"...

But since the goal here is simply to change line color, we can add a colorPicker that targets `.s_timeline_line` so we won't need to have a border-width value on it.